### ISSUE TITLE
FX graph mode quantization generic support for kwargs

### ISSUE THREAD

**Issue by vkuzo** [pytorch/pytorch#78117]
### 🐛 Describe the bug

FX graph mode quantization does not yet handle user using kwargs in all cases.  For example, this code snippet:

```
import torch
import torch.nn as nn
import torch.nn.functional as F
import torch.ao.quantize_fx as quantize_fx

class M(nn.Module):                                               
    def __init__(self):                                           
        super().__init__()                                        
        self.w = torch.randn(1, 1)                                
        self.b = torch.randn(1)                                   
                                                                  
    def forward(self, x):                                         
        x = F.linear(x, weight=self.w, bias=self.b)               
        return x                                                  
                                                                  
m = M().eval()                                                    
qconfig_dict = {'': torch.ao.quantization.default_qconfig}        
mp = quantize_fx.prepare_fx(m, qconfig_dict, (torch.randn(1, 1),))
mq = quantize_fx.convert_fx(mp)                                   
print(mq)                                                         
```

fails with error: https://gist.github.com/vkuzo/caa92fde3afe34e6a18e24162f62a1ae

Expected behavior here would be that specifying keyword arguments using both args and kwargs are supported.  There are additional cases to test here, such as `F.linear(x, bias=self=b, weight=self.w)`.

### Versions

'1.12.0a0+git5ebcb83'

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo

**Comment by jerryzh168** [comment]
probably similar to https://github.com/pytorch/pytorch/issues/73463. yeah we should fix this. maybe by normalizing the kwargs
